src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -553,6 +554,15 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
 
                 request.Metadata.Bindings.Add(binding.Name, bindingInfo);
             }
+
+            foreach (var property in metadata.Properties)
+            {
+                // worker properties are expected to be string values
+                request.Metadata.Properties.Add(property.Key, property.Value.ToString());
+            }
+
+            _workerChannelLogger?.LogDebug($"Adding {request.Metadata.Properties.Count} worker properties for {functionId}");
```

Comment:
Please use structured logging

Reply:
Have removed the log statement as it wasn't adding much value